.lsan-suppressions

```diff
@@ -1,2 +1,3 @@
 leak:ccrng_cryptographic_init_once
-leak:ccrng_cryptographic_generate
\ No newline at end of file
+leak:ccrng_cryptographic_generate
+leak:CRYPTO_zalloc
```

Comment:
Do you know why `CRYPTO_zalloc` reports a 56 byte leak on SLES 15? My guess is this is a bug in the version of OpenSSL on SLES 15. Either way, I think there is little to no risk this points to a bug in libmongocrypt. I do not think it is necessary to determine the root cause.